doc/endpoints.md

```diff
@@ -487,3 +495,73 @@ Removes an attendee.
     ]
 }
 ```
+
+# leaderboard
+## GET /
+```json
+{
+    "data": [
+        {
+            "avatar": "/uploads/attendee/avatars/ee4514b5-6b71-44ff-b26f-8afc0b8c7e51/original.png?v=63705800808",
+            "badges": [
+                {
+                    "avatar": "/uploads/badge/avatars/5/original.png?v=63705802360",
+                    "begin": "2019-02-12T00:00:00.000000Z",
+                    "description": "hackerino",
+                    "end": "2019-02-13T00:00:00.000000Z",
+                    "name": 5
+                }
+            ],
+            "id": "ee4514b5-6b71-44ff-b26f-8afc0b8c7e51",
+            "nickname": "Nick"
+        }
+    ]
+}
+```
+
+# Association
+## /:discord_id (Show)
+Fetches the id of the attendee associated with the given discord id
+
+### Valid: 
+```JSON
+{
+  "id": "f90b60ce-219c-4edc-a92a-e6c63fec4a4d"
+}
+```
+
+### Errors:
+
+- Discord id is not associated:
+
+```JSON 
+{
+  "error": "No attendee with that discord_id"
+}
+```
+
+## POST /
+
+```json
+{
+	"discord_association_code": "733405df-0b7b-4dc3-9666-3038fde44698",
+	"discord_id": "abc"
+}
+```
+
+### Valid: 
+```JSON
+{
+  "association": "participante"/"empresa"/"orador"/"staff"
+}
+```
```

Comment:
How did this get past the PR in Portuguese? 😆

Reply:
Well, you've got to talk to the discord team, their choice, not mine ahaha